pkg/client/daemon/tunrouter.go

```diff
@@ -191,10 +191,7 @@ func (t *tunRouter) setOutboundInfo(ctx context.Context, mi *daemon.OutboundInfo
 		defer cancel()
 
 		var conn *grpc.ClientConn
-		conn, err = grpc.DialContext(tc, fmt.Sprintf("127.0.0.1:%d", mi.ManagerPort),
-			grpc.WithInsecure(),
-			grpc.WithNoProxy(),
-			grpc.WithBlock())
+		conn, err = client.DialSocket(tc, client.ConnectorSocketName)
```

Comment:
Well, in that commit it does, but in the PR as a whole it doesn't.  Previously it was going through 1 hop: `kubectl port forward`.  Now it's going through 1 hop: the connector.

- before: daemon → kpf → cluster
- middle (8f59b2af8ecb39dba17a13851ad001927f06cf93): daemon → connector → kpf → cluster
- after (7cbbdf703a22191c9c1335c11d3bb6ef43bfe2c8): daemon → connector → cluster

Reply:
Now, there is an argument to be made that `service_proxy.go` operating at L7 isn't going to perform as well as operating at L4.  Maybe.  But I'd guess we're actually winning by being able to share more SPDY connections.  But also we can improve this if we want to get hacky treat requests to the manager service as opaque requests instead of structured data.

Reply:
Would it be possible to have the daemon do embedded port-forwards too so that we get daemon → cluster ?

Reply:
Because there might be auth extensions that get shelled out to by the kubeconfig, we really want communication with the kube stuff to happen as the normal user and not as root.

Reply:
Ah, yes. That makes sense.